etcd-backup-cronjob-monitor.PrometheusRule.yaml

```diff
@@ -21,5 +20,6 @@ spec:
     - alert: EtcdBackupCronJobStatusFailed
       expr: |
         kube_job_status_succeeded{namespace="etcd-backup"} == 0
+      for: 24h
```

Comment:
i'm this will really check if the last backup is older than 24h, shouldn't the check be on like the age of the last backup or something?

i'm really not sure about this and would need to dig deeper to be sure

Reply:
right, but the question is do we want to go down there? right @paraenggu